MUICore/setup.py

```diff
@@ -49,27 +49,11 @@ def finalize_options(self):
 
     def run(self):
         import shiv.cli
-
-        shiv.cli.main.callback(*shiv_args.values())
-
-
-class CopyBinaryCommand(Command):
```

Comment:
cyclic dependency with `build_binary` since `shiv` calls all commands in `cmdclass`. but as per the thoughts above i suppose it's not suited to be in MUICore in the long run